lib/standard/formatter.rb

```diff
@@ -66,12 +66,18 @@ def print_todo_warning
       todo_ignore_files = options[:todo_ignore_files]
       return unless todo_ignore_files
 
-      output.print <<-HEADER.gsub(/^ {8}/, "")
-        WARNING: this project is being migrated to standard gradually via `#{todo_file}` and is ignoring these files:
-      HEADER
-
-      todo_ignore_files.each do |f|
-        output.printf("  %s\n", f)
+      if todo_ignore_files.size.zero?
+        output.print <<-HEADER.gsub(/^ {10}/, "")
+          WARNING: this project appears to already be migrated to standard! If that is true, `#{todo_file}` can be removed. Otherwise, files will need added to the `ignore` list inside `#{todo_file}`.
```

Comment:
> files will need added to the `ignore`

This phrase confuses me a bit -- if zero violations occur shouldn't you be removing files from the ignore list (to generate more todos), if any?

Reply:
From my understanding, if someone wants to ignore certain files, then they need to add those files to the list. The idea behind the second part of the message is to capture directing the user on what to do next after generating the todo file.

Reply:
Ah, I see -- but what message should/will print if `standard` exits cleanly and this file also still exists? Shouldn't it just be "delete the file"?

Reply:
By clean exit, do you mean the situation where rubocop finds no issues? In that case, I think we could safely delete the file for them. The one caveat I'm considering, however, is to potentially keep the file and print the warning _unless_ `--fix` is applied. If `--fix` is provided then standard could remove the file.

With that said, `--fix` is intentionally changing project code whereas `.standard_todo.yml` isn't exactly "project-specific" in the same way. It's more a metadata file for standard. I can see an argument both ways.

What are your thoughts?